app/features-json/setting_json_controller.rb

```diff
@@ -0,0 +1,42 @@
+require_relative "api_controller"
+
+module FastlaneCI
+  # Controller for providing all data relating to system settings
+  class SettingJSONController < APIController
+    HOME = "/data/settings"
+
+    get HOME do
+      return json(Services.setting_service.settings)
+    end
+
+    post "#{HOME}/:setting_key" do
```

Comment:
I couldn't test this locally for now, as I'm at home and can't get auth to work. Also, I don't think we have a way to add tests for authenticated API endpoints yet